kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/logging/Logging.kt

```diff
@@ -67,6 +67,9 @@ class Logging {
       val rootLogger = logger(Logger.ROOT_LOGGER_NAME)
       val rootLevelCfg = Level.valueOf(System.getenv("LOG_LEVEL") ?: DEFAULT_LOG_LEVEL)
       rootLogger.level = rootLevelCfg
+
+      // Explicitly set log level for grpc-netty
+      logger("io.grpc.netty").level = Level.INFO
```

Comment:
I think make this WARN, half of the logs are coming through as INFO:

![image](https://github.com/TBD54566975/ftl/assets/41767/ea06cc2b-a35a-405b-b1d6-18dce1bc7dbf)

Which, related, looks like it's not being logged at the correct level for some reason. It looks like it's not using the custom JSON log formatter.

Reply:
i _think_ this change should also fix the JSON formatting, the `logger("io.grpc.netty")` invocation explicitly adds an appender for this class so should force it to inherit the root logging configs that it doesn't seem to be currently for some reason

Reply:
Ah nice!